schema/herocard-response-schema.json

```diff
@@ -33,7 +33,7 @@
     "link": {
       "type": "object",
       "properties": {
-        "href": {"type": "uri"}
```

Comment:
This was wrong: http://json-schema.org/draft-06/json-schema-release-notes.html#formats-uri-vs-uri-reference